### ISSUE TITLE
Schema migration fails: "Found non-empty schema"

### ISSUE THREAD

**Issue by copyandexecute** [A248/LibertyBans#31]
Hi, I wanted to use LibertyBans but somehow this super weird error occurs 
WHAT IS FLYAWAY?

![grafik](https://user-images.githubusercontent.com/58366920/115984670-9f432a80-a5a8-11eb-8115-849d775b5639.png)

**Comment by OskarZyg** [comment]
That just looks like a database adapter

**Comment by A248** [comment]
@copyandexecute , did you manage to resolve this problem? I have never encountered it myself. Please provide more information on your environment and database.

Flyway is a library used to perform schema updates in a consistent manner. In layman's terms, that means it updates the format the data is stored in. The first time LibertyBans runs, it sets up the database using flyway.
